rest-api-spec/src/main/resources/rest-api-spec/api/security.get_service_tokens.json

```diff
@@ -0,0 +1,34 @@
+{
+  "security.get_service_tokens":{
```

Comment:
I think this one is mismatched - The endpoint is for `/credential/` but the API is specifically named about tokens. I think it should be named  `get_service_credentials` instead.

Reply:
I changed it to `get_service_credentials`. Should the Java code also be updated for consistency and correctness, e.g. rename `GetServiceAccountTokensAction` to `GetServiceAccountCredentialsAction`? I can raise a separate PR just for renames and it should be trivial.

Reply:
I raised https://github.com/elastic/elasticsearch/pull/71917 for it.
It has renames discussed here and also another rename that I wanted to have for a while.